contracts/Permissioned.sol

```diff
@@ -234,6 +234,22 @@ contract Permissioned is ERC165, ERCTBDStorage {
         return false;
     }
 
+    function lockId(bytes32 _arg0, bytes32 _arg1) internal pure returns(bytes32) {
```

Comment:
Please add a comment to explain what this function set should be used for.

Reply:
done: https://github.com/daostack/access_control/pull/34/commits/776cc25c341aef6589ed0e2aa4f13774e986e668